DigitalOcean.API/Clients/LoadBalancerClient.cs

```diff
@@ -57,15 +57,15 @@ public sealed class LoadBalancerClient : ILoadBalancerClient {
 			return _connection.ExecuteRaw("load_balancers/{id}/droplets", parameters, dropletIds, Method.DELETE);
 		}
 
-		public Task AddForwardingRule(int loadBalancerId, ForwardingRules forwardingRules){
+		public Task AddForwardingRules(int loadBalancerId, Models.Requests.ForwardingRulesList forwardingRules){
```

Comment:
It seems Load Balancer ids are actually of type `string`.

Reply:
This is one of the few clients I haven't hand tested. Can't believe didn't notice that one at all. I'll swap them all next.

Reply:
Done.